CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # osquery Changelog
 
+<a name="5.7.0"></a>
+## [5.7.0](https://github.com/osquery/osquery/releases/tag/5.7.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.6.0...5.7.0)
+
+Representing commits from 12 contributors! Thank you all.
+
+### CVEs
+
+Addressed by updating a library:
+
+Ignored due to not affecting osquery:
+  - libzstd CVE-2021-24031 via ([#7865](https://github.com/osquery/osquery/pull/7865))
+
+### New Features
+
+- New table `security_profile_info` to retrieve security profile information on Windows ([#7794](https://github.com/osquery/osquery/pull/7794))
+
+### Table Changes
+
+- Add column to `es_process_events` for process codesigning flags ([#7726](https://github.com/osquery/osquery/pull/7726))
+- Shimcache: Only check CurrentControlSet to avoid duplicate rows ([#7832](https://github.com/osquery/osquery/pull/7832))
+- processes: Fix the procfs memory unit kB, which is 1024 bytes not 1000 ([#7818](https://github.com/osquery/osquery/pull/7818))
+- Fix permissions on opening pipes for reading in `pipes` table ([#7810](https://github.com/osquery/osquery/pull/7810))
+- Fix the empty `host` column from `logged_in_users` table ([#7685](https://github.com/osquery/osquery/pull/7685))
+- docker_containers: Don't report finished_at for a container which is still running ([#7783](https://github.com/osquery/osquery/pull/7783))
+- `processes`: Stabilize the `start_time` column value on macOS and Linux ([#7788](https://github.com/osquery/osquery/pull/7788))
+
+### Bug Fixes
+
+- Do not access the AWS SDK request content type if missing ([#7834](https://github.com/osquery/osquery/pull/7834))
+- Fix deadlock when logging happens during a database reset ([#7798](https://github.com/osquery/osquery/pull/7798))
+- Fix handling of some errors during an AWS HTTP request ([#7811](https://github.com/osquery/osquery/pull/7811))
+
+### Documentation
+
+- CHANGELOG 5.6.0 ([#7804](https://github.com/osquery/osquery/pull/7804))
+- Add link to official YARA docs ([#7792](https://github.com/osquery/osquery/pull/7792))
+- Fixing typo in keychain_items ([#7790](https://github.com/osquery/osquery/pull/7790))
```

Comment:
"Fix" (to be consistent with other items)

Also, we could change "keychain_items" to "`keychain_items`"

Reply:
Thanks! I've also added some more quoting on other table and column names.